nomt/src/beatree/mod.rs

```diff
@@ -108,6 +110,7 @@ impl Tree {
             bbn_store,
             primary_staging: BTreeMap::new(),
             secondary_staging: None,
+            leaf_cache: leaf_cache::LeafCache::new(32, 1 << 16),
```

Comment:
Those numbers look magic-y. I assume this is/will be changed in the following PRs.

Reply:
Definitely, just like the merkle page cache we need to add this to the public API